### ISSUE TITLE
shoutmehindi.com

### ISSUE THREAD

**Issue by mediumkreation** [mediumkreation/IndianList#6]
Below seems to be an Adserve, but is represented as content, as there is an intermediate content page. should this be removed ?
Should we remove it only at the external(redirecting) link ? or should the images and other references also be removed.

![shoutmehindi com_18-07-30 21 12 52](https://user-images.githubusercontent.com/39060814/43469660-4c854b32-9504-11e8-927a-fbed163d2175.png)

![shoutmehindi com_18-07-31 21 07 37](https://user-images.githubusercontent.com/39060814/43470559-7a6d4106-9506-11e8-90a1-3239097f5fc5.png)

**Comment by monzta** [comment]
I would say it should stay since it seems to be part of the site.

**Comment by mediumkreation** [comment]
should this not be removed .. as this is redirecting to external site
**Found in url**: https://shoutmehindi.com/deal/hostgator-india-hosting-discount-coupon/
![shoutmehindi com_18-07-31 21 07 37](https://user-images.githubusercontent.com/39060814/43470559-7a6d4106-9506-11e8-90a1-3239097f5fc5.png)

**Suggested Filter**
```
shoutmehindi.com##.coupon_dealtop
```


secondly, there are inline links too.. should I remove the links withe below filter..
![shoutmehindi com_18-08-04 18 47 28](https://user-images.githubusercontent.com/39060814/43676961-2c29306c-9818-11e8-9261-fb13f83083d7.png)
**Suggested Filter**
```
shoutmehindi.com##a[href*="go/HostgatorIndia"]
```

**Comment by mediumkreation** [comment]
resolved by PR [#15](https://github.com/mediumkreation/IndianList/pull/15)
